src/app/tracing/graph/cy-graph/virtual-zoom-utils.ts

```diff
@@ -28,6 +28,16 @@ export function getAvailableSpace(htmlElement: HTMLElement): Size {
     };
 }
 
+export function unzoomedToZoomedModelPosition(
+    unzoomedModelPosition: Position,
```

Comment:
I know this was my doing, but please change
```
export function unzoomedToZoomedModelPosition(
    unzoomedModelPosition: Position
```
to
```
export function zoomedToUnzoomedModelPosition(
    zoomedModelPosition: Position
```

Background: The VirtualZoomGraph knows the true modelPositions but supplies the InteractiveGraph with the zoomed Positions. On events we get the zoomed Positions and therefore they need to get unzoomed.